fdbclient/NativeAPI.actor.cpp

```diff
@@ -8776,6 +8780,7 @@ Future<DatabaseSharedState*> DatabaseContext::initSharedState() {
 }
 
 void DatabaseContext::setSharedState(DatabaseSharedState* p) {
+	ASSERT(networkOptions.disableBypass);
```

Comment:
At the native level, we could throw errors (but not asserts) when the user sets the `USE_GRV_CACHE` option. We may need to API version guard that behavior.